builder/docker/config.go

```diff
@@ -88,6 +99,9 @@ type Config struct {
 	// If true, the configured image will be pulled using `docker pull` prior
 	// to use. Otherwise, it is assumed the image already exists and can be
 	// used. This defaults to true if not set.
+	//
+	// If using `build`, this will not be honoured, as the `pull` option for
```

Comment:
Nit: I think this is a bit more of a simple way to describe this
```suggestion
	// If using `build`, this will field be ignored, as the `pull` option for
```